FreeRTOS/FreeRTOS
# Add MQTT Mutual Auth wolfSSL Demo

Add MQTT Mutual Auth wolfSSL Demo

Description
-----------
This demo shows an MQTT subscriber / publisher establishing a TLS connection with an MQTT broker to send and receive topics in encrypted form. wolfSSL acts as a TLS library under the FreeRTOSMQTT library. Also, an actual AWS MQTT broker endpoint is used.

Test Steps
-----------
1. Double-click the solution file named "mqtt_mutual_auth_demo_wolfSSL.sln" included in this folder to start Visual Studio.
2. Build the RTOSDemo project
3. Run the RTOSDemo.exe with debugger

Related Issue
-----------
None.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
